Content.Server/Atmos/EntitySystems/FlammableSystem.cs

```diff
@@ -317,9 +334,19 @@ public override void Update(float frameTime)
 
                 if(transform.GridUid != null)
                 {
+                    var tile = _transformSystem.GetGridOrMapTilePosition(uid, transform);
+                    var temperature = 700f;
+                    var volume = 50f;
                     _atmosphereSystem.HotspotExpose(transform.GridUid.Value,
-                        _transformSystem.GetGridOrMapTilePosition(uid, transform),
-                        700f, 50f, true);
+                        tile,
+                        temperature, volume, true);
+
+                    // Spread the fire to other flammables on the same tile
+                    var fireEvent = new TileFireEvent(temperature, volume);
+                    foreach (var entity in _lookup.GetEntitiesIntersecting(transform.GridUid.Value, tile))
+                    {
+                        RaiseLocalEvent(entity, ref fireEvent, false);
+                    }
 
```

Comment:
This looks very suss. Instead of the flammable relaying itself to atmos it's relaying directly to other flammables in an expensive loop every tick.